pkg/commands/compute/compute_test.go

```diff
@@ -252,6 +228,32 @@ func TestGetIgnoredFiles(t *testing.T) {
 }
 
 func TestGetNonIgnoredFiles(t *testing.T) {
+	// We're going to chdir to a build environment,
```

Comment:
The changes to this function can be summarised as follows...

Define a single temp environment rather than having a new environment for each test scenario.